api/src/pcapi/core/mails/transactional/educational/eac_finance_incident_notification.py

```diff
@@ -0,0 +1,33 @@
+from pcapi.core import mails
+from pcapi.core.finance import models as finance_models
+from pcapi.core.mails import models as mails_models
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+
+
+def send_eac_finance_incident_emails(
+    finance_incident: finance_models.FinanceIncident,
+) -> bool:
+    offer = finance_incident.booking_finance_incidents[0].collectiveBooking.collectiveStock.collectiveOffer
+    venue = finance_incident.venue
+
+    if not venue.current_reimbursement_point or not venue.current_reimbursement_point.bookingEmail:
+        return True
+
+    offers = []
+    for booking_finance_incident in finance_incident.booking_finance_incidents:
+        offer = booking_finance_incident.booking.stock.offer
```

Comment:
[bug ?] ll me semble qu'il y a une erreur, dans cette fonction qui traite les incidents de finance portant sur des réservations **collectives** : 

```suggestion
        offer = booking_finance_incident.collectiveBooking.collectiveStock.collectiveOffer
```